Common/Data/Auxiliary/FactorFileRow.cs

```diff
@@ -36,12 +40,20 @@ public class FactorFileRow
         /// <summary>
         /// Gets the price factor associated with this data
         /// </summary>
-        public decimal PriceFactor { get; private set; }
+        public decimal PriceFactor
+        {
+            get { return _priceFactor.RoundToSignificantDigits(7); }
```

Comment:
Just to preserve as much significance as possible for the triple leveraged ETF's deep splitting cases lets remove rounding.